src/TodoListItem.js

```diff
@@ -1,8 +1,9 @@
 import React from "react";
+import style from "./TodoListItem.module.css";
 
 function TodoListItem({ todo, onRemoveTodo }) {
     return (
-        <li>{todo.title} <button onClick={() => onRemoveTodo(todo.id)}>Remove</button></li>
+        <li className={style.ListItem}>{todo.title} <button onClick={() => onRemoveTodo(todo.id)} className={style.button}>Remove</button></li>
```

Comment:
Good job naming your styles after your components, this will be important as your app grows.